codalab/apps/web/views.py

```diff
@@ -506,12 +510,39 @@ def get_context_data(self, **kwargs):
             if self.request.user.is_authenticated() and self.request.user in [x.user for x in all_participants]:
                 context['my_status'] = [x.status for x in all_participants if x.user == self.request.user][0].codename
                 context['my_participant'] = competition.participants.get(user=self.request.user)
+                user_team = get_user_team(context['my_participant'], competition)
+                context['my_team'] = user_team
                 phase_iterator = iter(all_phases)
                 for phase in phase_iterator:
                     submissions[phase] = models.CompetitionSubmission.objects.filter(participant=context['my_participant'], phase=phase)
                     if phase.is_active:
                         context['my_active_phase_submissions'] = submissions[phase]
                 context['my_submissions'] = submissions
+
+                # Add alert counters for admins
+                context['comp_num_pending_teams'] = len(get_competition_pending_teams(competition))
+
+                if user_team is None:
+                    context['new_team_submission'] = 0
+                    context['my_team_alert'] = 0
+
+                    # Check if there are pending invitations from other teams
+                    if len(get_user_requests(context['my_participant'], competition)) > 0:
+                        context['my_team_alert'] = 1
+                else:
+                    context['new_team_submission'] = len(get_last_team_submissions(context['my_team'], 1))
+                    if context['my_team'].creator == self.request.user:
+                        context['my_team_alert'] = 0
+                        # Check if my team has changed in the last 24 hours (accepted/edited)
+                        if user_team.last_modified >= timezone.now() - timedelta(1):
+                            context['my_team_alert'] = 1
+                        else:
+                            # Check if there are pending requests from other users
+                            if len(get_team_pending_membership(user_team)) > 0:
+                                context['my_team_alert'] = 1
+                    else:
+                        context['my_team_alert'] = 0
+                        # TODO: Check if my membership changed in the last 24h
```

Comment:
I can be improved. Right now just some "easy to detect" changes are used in order to show the alert on the Team tab